tcp/options.ml

```diff
@@ -16,6 +16,8 @@
 
 (* TCP options parsing *)
 
+open Sexplib.Std
```

Comment:
better in what sense? I just want to bring the `sexp_of_int` and other basic converters in scope.